src/server/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "secure": true
+} 
```

Comment:
Doesn't `.gitignore` say to ignore this file?

Reply:
Yes, but it also says to ignore a theoretical `src/client/config.json`. You can list the `/src/server/config.json` path in `.gitignore` to be more specific.